packages/server/routes/api/users.js

```diff
@@ -53,4 +54,42 @@ router.get(
   }
 );
 
+// Get all guilds for a specific user
+router.get(
+  "/:userId/guilds",
+  AuthController.authenticate,
+  async (request, response) => {
+    try {
+      const { userId } = request.params;
+
+      // Check if the user exists
+      const user = await UserController.getUser(userId);
+
+      if (!user) {
+        return response.status(404).json({
+          status: "fail",
+          data: {
+            userId: `User not found with ID ${userId}`,
+          },
+        });
+      }
```

Comment:
Like I said in earlier comment, we'll be using [express-validator](These checks won't be necessary. I'm getting all of us to migrate over to express-validator to streamline error handling. I'll tell you more details in Discord but you can start by reading docs [here](https://express-validator.github.io/docs/guides/getting-started#adding-a-validator).) for error handling now. Eventually, these checks will be replaced by an express validator instead.